src/operations/blas3/gemm_local_joint_matrix.hpp

```diff
@@ -62,6 +62,7 @@ namespace blas {
  * @tparam TransA  iff true, matrix A will be transposed on the fly
  * @tparam TransB  iff true, matrix B will be transposed on the fly
  * @tparam element_t  type of matrix elements
+ * @tparam element_out_t  type of output matrix elements (C) and scalars
```

Comment:
I'd also like to see `element_t` renamed to `element_in_t`.

Reply:
The extra template parameter has been removed.